apps/meteor/app/lib/server/functions/setUsername.ts

```diff
@@ -67,78 +68,81 @@ export const setUsernameWithValidation = async (userId: string, username: string
 	void notifyOnUserChange({ clientAction: 'updated', id: user._id, diff: { username } });
 };
 
-export const _setUsername = async function (userId: string, u: string, fullUser: IUser, updater?: Updater<IUser>): Promise<unknown> {
-	const username = u.trim();
+export const _setUsernameWithSession = (session?: ClientSession) =>
```

Comment:
I'm not sure about suffix this with `withSession` since the session is not required.

Reply:
I did currying here to avoid adding extra parameters to the function itself. I exported `_setUsername = _setUsernameWithSession()` below so that old references to this function still work as they did.

The parameter is indeed not required, but you should only call this method if you plan on using the session. Otherwise the "old" function is still exported under the same name it had before.